app/experimenter/experiments/models.py

```diff
@@ -410,14 +433,20 @@ def proposed_end_date(self):
 
     @property
     def computed_enrollment_days(self):
-        paused_change = (
-            self.changes.all()
-            .filter(NimbusChangeLog.Filters.IS_APPROVED_PAUSE)
-            .order_by("changed_on")
-            .first()
-        )
-        if paused_change:
+        changes = self.changes.all()
+        paused_changelogs = [
+            c
+            for c in changes
+            if c.experiment_data["is_paused"]
+            and c.new_status == NimbusExperiment.Status.LIVE
+            and c.new_status_next is None
+            and c.new_publish_status == NimbusExperiment.PublishStatus.IDLE
+        ]
+
+        if paused_changelogs:
+            paused_change = sorted(paused_changelogs, key=lambda c: c.changed_on)[-1]
```

Comment:
what's the `-1` doing here? popping the changelog?

Reply:
so `sorted` returns the list, and [-1] meaning returning the last element of the list, so based on changes made, sorting based on changed on and returning results 🚀